types/theo/index.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for theo 8.1
+// Type definitions for theo 8.1.1
```

Comment:
Updated to match supported package release: https://github.com/salesforce-ux/theo/releases/tag/v8.1.1